include/tilt/base/type.h

```diff
@@ -158,10 +158,10 @@ struct Type {
     const Iter iter;
 
     Type(DataType dtype, Iter iter) :
-        dtype(move(dtype)), iter(iter)
+        dtype(std::move(dtype)), iter(iter)
```

Comment:
There might be a way to suppress this warning. Currently I have set to throw most of the warning as Error. If you can suppress it, you can use that.

If not, can you remove the `using namespace std` at the beginning of all the source/header files?